server/db/models/borrowed_books.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable camelcase */
```

Comment:
Please rebase off develop. This model already exists under "BorrowsBooks.js"

Reply:
Done